demosplan/DemosPlanCoreBundle/Entity/User/Customer.php

```diff
@@ -191,19 +191,27 @@ class Customer extends CoreEntity implements UuidEntityInterface, CustomerInterf
     #[Assert\Length(max: 65536)]
     protected $overviewDescriptionInSimpleLanguage = '';
 
+    /**
+     * @var Collection<int, SupportContact>
+     *
+     * @ORM\OneToMany(targetEntity="SupportContact", mappedBy="customer")
+     */
+    protected Collection $contacts;
+
     public function __construct(/**
      * @ORM\Column(name="_c_name", type="string", length=50, nullable=false)
      */
-    private string $name, /**
+        private string $name, /**
      * @ORM\Column(name="_c_subdomain", type="string", length=50, nullable=false)
      */
-    private string $subdomain, string $mapAttribution = '')
+        private string $subdomain, string $mapAttribution = '')
```

Comment:
wrong indentation

Reply:
wasnt me - and it looks strange as the orm Params are within the __construct method as a php8 private-property-param